src/test/feature/vexecutor/test_vexecutor.cpp

```diff
@@ -60,11 +59,47 @@ TEST_F(TestVexecutor, vadt)
 
 	// run sql file to get ans file and then diff it with out file
 	util.execSQLFile("vexecutor/sql/create_type.sql",
-	                 "vexecutor/ans/create_type.ans");
+					 "vexecutor/ans/create_type.ans");
 
 	util.execSQLFile("vexecutor/sql/vadt.sql",
-	                 "vexecutor/ans/vadt.ans");
+					 "vexecutor/ans/vadt.ans");
 
 	util.execSQLFile("vexecutor/sql/drop_type.sql",
-	                 "vexecutor/ans/drop_type.ans");
+					 "vexecutor/ans/drop_type.ans");
 }
+
+TEST_F(TestVexecutor, scanframework)
+{
+	hawq::test::SQLUtility util;
+
+	util.execute("drop table if exists test1");
+	util.execute("create table test1 ("
+				 "	unique1		int4,"
+				 "	unique2		int4,"
+				 "	two			int4,"
+				 "	four		int4,"
+				 "	ten			int4,"
+				 "	twenty		int4,"
+				 "	hundred		int4,"
+				 "	thousand	int4,"
+				 "	twothousand	int4,"
+				 "	fivethous	int4,"
+				 "	tenthous	int4,"
+				 "	odd			int4,"
+				 "	even		int4,"
+				 "	stringu1	name,"
+				 "	stringu2	name,"
+				 "	string4		name) WITH (appendonly = true, orientation = PARQUET, pagesize = 1048576, rowgroupsize = 8388608, compresstype = SNAPPY) DISTRIBUTED RANDOMLY;");
+
+  std::string pwd = util.getTestRootPath();
+  std::string cmd = "COPY test1 FROM '" + pwd + "/vexecutor/data/tenk.data'";
```

Comment:
why need so many data in tenk.data?

Reply:
I believe that it can relieve future test workload.